src/buffer/out/textBuffer.cpp

```diff
@@ -2137,6 +2138,15 @@ std::string TextBuffer::GenHTML(const TextAndColor& rows,
                     htmlBuilder << "background-color:";
                     htmlBuilder << Utils::ColorToHexString(bkColor.value());
                     htmlBuilder << ";";
+                    htmlBuilder << "text-decoration:";
+                    htmlBuilder << textProps.IsUnderlined();
+                    htmlBuilder << ";";
+                    htmlBuilder << "font-weight:";
+                    htmlBuilder << textProps.IsBold();
+                    htmlBuilder << ";";
+                    htmlBuilder << "font-style:";
+                    htmlBuilder << textProps.IsItalic();
+                    htmlBuilder << ";";

```

Comment:
I'm somewhat doubtful that this works correctly, because these 3 CSS properties don't accept true/false as their arguments.

Reply:
Also note that the `textProps` variable doesn't seem to be filled in with anything, and we'd need to be tracking the *changes* to these styles, the same way we're tracking changes to the colors, so we're only writing out a new span when necessary.